SourceKitLSPDevUtils/Sources/ConfigSchemaGen/OptionSchema.swift

```diff
@@ -234,4 +269,52 @@ struct OptionSchemaContext {
     }
     return docLines.joined(separator: " ")
   }
+
+  private static func extractDiscriminatorFieldName(_ trivia: Trivia) -> String? {
+    let docLines = trivia.flatMap { piece -> [Substring] in
+      switch piece {
+      case .docBlockComment(let text):
+        assert(text.hasPrefix("/**") && text.hasSuffix("*/"), "Unexpected doc block comment format: \(text)")
+        return text.dropFirst(3).dropLast(2).split { $0.isNewline }
+      case .docLineComment(let text):
+        assert(text.hasPrefix("///"), "Unexpected doc line comment format: \(text)")
+        let text = text.dropFirst(3)
+        return [text]
+      default:
+        return []
+      }
+    }
+
+    for line in docLines {
+      var trimmed = line
+      while trimmed.first?.isWhitespace == true {
+        trimmed = trimmed.dropFirst()
+      }
```

Comment:
I think this could just be 

```suggestion
      var trimmed = line.drop(while: \.isWhitespace)
```